### ISSUE TITLE
Add support for aged cache directories

### ISSUE THREAD

**Issue by holdenk** [filecoin-project/saturn-node#131]
Eg on some servers there is X TB of NVMe and Y TB of SSD and K TB of HDD and it would be nice to have expiration just slowly move data down the stack in terms of speed.

**Comment by DiegoRBaquero** [comment]
I don't think this is configurable on nginx

**Comment by vorburger** [comment]
> slowly move data down the stack in terms of speed

+1; this thought has crossed my mind as well.

> I don't think this is configurable on nginx

It's probably just something we would have to "build" (as in, code out, with "domain knowledge"), not something Nginx could automagically "just do" for us with config.

**Comment by holdenk** [comment]
Yeah if theres interest in this maybe I'll take a crack it over the winter break, I've got a new server on with 8tb of NVMe and another 8tb of HDE I'd like to take advantage of (I know in practice right now the cache depth is likely to all fit inside the first 8tb of NVMe but you know the "future") :)

**Comment by vorburger** [comment]
@holdenk I suspect The Way To Go on this is probably https://man.archlinux.org/man/lvmcache.7 - WDYT?
